st/clitests/s3cmd_spec.py

```diff
@@ -31,7 +31,7 @@
 import s3kvs
 
 # Helps debugging
-# Config.log_enabled = True
+Config.log_enabled = True
```

Comment:
Do you want to disable this, if it was enabled for debugging?

Reply:
Done